docs/compute-element/submit-htcondor-ce.md

```diff
@@ -47,6 +47,9 @@ user@host $ condor_ce_trace --debug --attribute='+xcount=4' --attribute='+maxMem
 
 For a list of other attributes that can be set with the `--attribute` option, consult the [job attributes](#job-attributes) section.
 
+!!! warning
+    If the htcondor-ce is submitting to a non-htcondor based cluster, job attributes may not be correctly translated. 
+
```

Comment:
Reading over this I think maybe we should provide a note instead that says non-HTCondor batch systems may require additional configuration to use additional job attributes and link to [this section](http://opensciencegrid.org/docs/compute-element/job-router-recipes/#setting-batch-system-directives) of the router recipes.

Reply:
Could you change it from a `!!! warning` to a `!!! note`?